dom.bs

```diff
@@ -3040,6 +3040,34 @@ standards that want to define APIs shared between <a for=/>documents</a> and
  <a for=CharacterData>data</a> is the string and <a for=Node>node document</a> is
  <var>document</var>.
 
+ <li><p>Let <var>isScriptElement</var> be <var>parent</var> is a {{HTMLScriptElement}}.
+
+ <li><p><a for=set>For each</a> <var>value</var> in <var>nodes</var>:
```

Comment:
The formatting here is wrong. Please ensure it's consistent with the rest of the document. The README should have instructions or a pointer to them.

Reply:
Hopefully this is fixed.